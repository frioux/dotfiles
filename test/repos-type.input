#!/bin/bash

makefile1='test/,repos-type.Makefile1'
makefile2='test/,repos-type.Makefile2'

cat >"$makefile1" <<END
INSTALLATION_DIR :=# None
include Makefile
echo:
	@echo \$(REPOS_TYPE) >&2
END

cat >"$makefile2" <<END
INSTALLATION_DIR :=# None
REPOS_TYPE := vim-script
include Makefile
echo:
	@echo \$(REPOS_TYPE) >&2
END

make --always-make -f "$makefile1" >/dev/null echo
make --always-make -f "$makefile2" >/dev/null echo
make --always-make -f "$makefile1" >/dev/null echo REPOS_TYPE=amane
make --always-make -f "$makefile2" >/dev/null echo REPOS_TYPE=amane

rm -rf "$makefile1" "$makefile2"

# __END__
