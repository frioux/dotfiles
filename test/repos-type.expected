generic
vim-script
amane
amane
